### ISSUE TITLE
Mavic 2 Zoom does not save flight logs

### ISSUE THREAD

**Issue by andrewdittmer** [dji-sdk/Mobile-SDK-Android#1241]
Other drones store the .dat log files to the directory specified by `DJISDKManager.getInstance().logPath` in the `MCDatFlightRecords` directory.

Mavic 2 Zoom doesn't store any .dat files in that directory. This was observed on a Mavic 2 Zoom with firmware V01.00.0790 and MSDK 4.16.4

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/88181'>Zendesk ticket #88181</a>:<br><br>Can you find the flight controller logs under the specified path using the getFlycLogPath interface?
<br><p><em><small>°°°</small></em></p>

**Comment by andrewdittmer** [comment]
No, I can't see the logs there using getFlycLogPath when using Mavic 2 Zoom

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/88181'>Zendesk ticket #88181</a>:<br><br>Have you used the SDK for real flights? Please note that flight records (Flight) are not generated when using the simulator.
<br><p><em><small>°°°</small></em></p>
